STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year. For the 2020
+year, one seat is assigned to each of Google, IBM/RedHat, and VMware. The
+committee shall review this composition at least annually and update these
+rules.
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding conformance and scope.
```

Comment:
What about decisions regarding assignment of the mark? What ability does the trademark committee have to influence those decisions?

Reply:
I would also like to understand:

- what responsibility the TM committee has to promptly review PRs that affect API scope and conformance
- what qualifications members of this committee must have

It seems like anyone on this committee has to have sufficient context to make thoughtful, informed decisions about the project and the needs of the vendor they represent.

Reply:
Explicitly added assignment of mark and charter to the set of veto-able decisions (which is to say, all decisions).